.circleci/e2e-testnet.sh

```diff
@@ -3,5 +3,6 @@
 docker run --rm -ti \
     -e E2E_NO_DEPLOY=true \
     -e E2E_PUBLIC_API_ENDPOINT=$E2E_PUBLIC_API_ENDPOINT \
+    -e NETWORK_ID=54 \
```

Comment:
it becomes a string or number when the code parses it?

Reply:
It's always a string.